core.ts

```diff
@@ -897,7 +895,9 @@ abstract class PinejsClientCoreTemplate<
 			for (const validParam of validParams) {
 				const value = params[validParam];
 				if (value != null) {
-					this[validParam] = value;
+					(this[validParam] as PinejsClientCoreTemplate<
```

Comment:
Are the parentheses necessary here?

Reply:
Yes, otherwise it's a syntax error.
I assume that prettier would have already changed this if it thought that it should.